### PR TITLE
Fix module member names for importing with renaming.

### LINKED ISSUE #10956: [TypeChecker] ICE in ModuleType::nativeMembers because symbol name (alias) and exported symbol name are different
## Description

```
==== Source:s1.sol====
int constant a;
==== Source:s2.sol====
import {a as e} from "s1.sol";
import "s2.sol" as M;
contract C {
  function f() { (M.a); }
}
```

throws

https://github.com/ethereum/solidity/commit/9ca389d6cd82410bcc887fd42ab2318b90307608#diff-5cc133bfc337e9ca7fe7ee9842d5ddfa492fa559149cf8acec4a7d9ede50c9aaR3812

Introduced by https://github.com/ethereum/solidity/pull/10880

## Steps to Reproduce

1. Copy test file to `test/libsolidity/syntaxTests/multiSource/import_alias.sol`

2.
```
$ isoltest -t syntaxTests/multiSource/import_alias
```

### PR DESCRIPTION
**Pull request by chriseth**
Fixes https://github.com/ethereum/solidity/issues/10956

### CODE REVIEW

**Comment by chriseth** [comment]
Not sure why the tests are failing. Will try locally.

File: test/libsolidity/syntaxTests/multiSource/import_alias.sol [hunk 1]
```diff
@@ -0,0 +1,8 @@
+==== Source:s1.sol====
```
Comment (cameel) [inline test/libsolidity/syntaxTests/multiSource/import_alias.sol]: https://github.com/ethereum/solidity/pull/10993#issuecomment-784109217

> Not sure why the tests are failing. Will try locally.

Maybe it's just that the file splitter expects whitespace here?

```suggestion
==== Source: s1.sol ====
```

File: test/libsolidity/syntaxTests/multiSource/import_alias.sol [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+int constant a = 2;
+==== Source:s2.sol====
+import {a as e} from "s1.sol";
+import "s2.sol" as M;
+contract C {
+  function f() public pure returns (int) { return M.e; }
```
Comment (cameel) [inline test/libsolidity/syntaxTests/multiSource/import_alias.sol:7]:
```suggestion
    function f() public pure returns (int) { return M.e; }
```

**Review by cameel — COMMENTED** [review]
The fix looks good but the weird thing is that I can't reproduce it by compiling files via `solc`. It only happens in tests.

**Review by cameel — APPROVED** [review]

**Pull request merged**